roles/custom/matrix-synapse-admin/defaults/main.yml

```diff
@@ -175,6 +175,9 @@ matrix_synapse_admin_configuration: "{{ matrix_synapse_admin_configuration_defau
 # restricts the homeserver(s), so that the user can no longer define a homeserver manually during login.
 matrix_synapse_admin_config_restrictBaseUrl: "{{ [matrix_homeserver_url] }}"  # noqa var-naming
 
+# Controls the supportURL configuration setting, which, if defined, replaces the default link to the Synapse Admin GitHub repository.
```

Comment:
This comment seems to be inaccurate.

Reply:
[why?](https://github.com/etkecc/synapse-admin/blob/main/src/components/AdminLayout.tsx#L4)